src/System.Net.HttpListener/src/System/Net/HttpListener.Windows.cs

```diff
@@ -102,104 +71,69 @@ internal SafeHandle RequestQueueHandle
             }
         }
 
-        public AuthenticationSchemeSelector AuthenticationSchemeSelectorDelegate
+        private void ValidateV2Property()
```

Comment:
Ah, then it's actually making things closer to NetFX which is what I wanted! Thanks for explaining!